### ISSUE TITLE
Consider pep8 style for being the default

### ISSUE THREAD

**Issue by johnthagen** [PyCQA/flake8-import-order#74]
Now that there is a `pep8` style, I would suggest that it be considered for the default.  While I personally use `google` and will continue to specify this, I think for the general use case this makes the most sense as the default.

I totally get though that this was made for the `cryptography` devs by the `cryptography` dev and that is clearly stated in the `README`.  So this is not a necessary change, but I do think it would be just a little better.  This would allow someone to just `pip install` it along side other PEP8 extensions (like `pep8-naming`) and get an out-of-the-box experience that they might expect.

Again, thanks for the great tool.

**Comment by pgjones** [comment]
I think this probably makes sense now the project is part of PyCQA, although I suspect it would break a many workflows. 
My preference is to change the default, although I'd like others to comment before acting.

**Comment by sigmavirus24** [comment]
On June 26, 2016 11:10:16 AM CDT, Phil Jones <notifications@github.com> wrote:
>I think this probably makes sense now the project is part of PyCQA,

Membership doesn't mean anything other than extra help with maintenance. :)

>although I suspect it would break a many workflows. 

That would make me biased against changing it, personally.


-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by johnthagen** [comment]
An option could be to make the change and release it as a new major version (1.0?) to indicate that a backwards incompatible change have been made.

**Comment by sigmavirus24** [comment]
> get an out-of-the-box experience that they might expect

I'm not sure I agree with this sentiment. Before Flake8 itself started using flake8-import-order, I took a close look at the plugin, read what you read about cryptography import style and provenance and the other import styles. I then configured appropriately after installing it and experimenting with it. Maybe I'm atypical, but the project name isn't pep8-import-order, it's flake8-import-order. There's no implication that you're following PEP-0008 import style guide just because of "Flake8" (at least not to me). There are too many other projects/plugins prefaced with "flake8-" for there to be any implication of "PEP-0008-ness" in any of them.
